docs/topics/what-is-the-toolkit.adoc

```diff
@@ -22,7 +22,11 @@ The {ProductName} ({ProductShortName}) is an extensible and customizable rule-ba
 * Migrating from Oracle WebLogic or IBM WebSphere Application Server to Red Hat JBoss Enterprise Application Platform
 * Containerizing applications and making them cloud-ready
 * Migrating from Java Spring Boot to Quarkus
-* Updating from Oracle JDK to OpenJDK

```

Comment:
@RichardHoch this migration path should still be valid and not to be removed.

right @PhilipCattanach ?

Reply:
@stillalearner -- lines 26 and 27 replace and specify line 25; that is we now specify 2 migration paths rather than have 1 general comment.

Reply:
Thanks for clarifying @RichardHoch

Reply:
@RichardHoch from what I understand, OracleJDK to OpenJDK is a different migration path than migrating from older OpenJDK to newer OpenJDK

right @sshveta / @PhilipCattanach ?